README.md

```diff
@@ -2,24 +2,11 @@
 
 > Simple git-hook integration for your gulp workflows.
 
-guppy streamlines and extends your git-hooks by integrating them with your 
-[gulp](http://gulpjs.com) workflow. This enables you to have **gulp tasks that 
-run when triggered by a git-hook**, which means you can do cool things like 
-abort a commit if your tests are failing. Git-hooks can now be managed through 
-[npm](https://npmjs.org), allowing them to automatically be installed and 
-updated. And because they integrate with gulp, it's easy to modify the workflow 
-and even combine hooks with your other gulp tasks.
```

Comment:
Motivation: No need to mention about workflow modification capability. Mentioned in various other occasions across the text.